python/semantic_kernel/connectors/ai/google_palm/services/gp_chat_completion.py

```diff
@@ -64,11 +62,9 @@ async def complete_chat_async(
                 else "I don't know."
                 for candidate in response.candidates
             ]
-        else:
-            if response.last is None:
-                return "I don't know."  # PaLM returns None if it doesn't know
-            else:
-                return response.last
+        if response.last is None:
+            return "I don't know."  # PaLM returns None if it doesn't know
```

Comment:
nit: not from this change but wouldn't it be better if this was a constant?

Reply:
Thanks, we can address in a next iteration.